src/test/java/HomeTest.java

```diff
@@ -0,0 +1,128 @@
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.Wait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+import pages.PlayListPage;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+
+public class HomeTest extends BaseTest {
+
+    String newPlayListName = "Sample Edited Playlist";
+
+    @Test
+    public void HoverOverPlayBtnAndPlaySong() throws InterruptedException {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        loginPage.login();
+        Thread.sleep(5000);
+        hoverPlay();
+    //  doubleClickPlayList();
+    //  Thread.sleep(2000);
+    //  Assert.assertTrue(hoverPlay().isDisplayed());
+    }
+
+    @Test
+    public void countSongsInPlayList() throws InterruptedException {
+
+        //Choose a Playlist by Name
+        choosePlayListByName("TestPro Playlist");
+        //DisplayAllSongs
+        displayAllSongs();
+        //Number of Songs are equal to number of songs displayed in the info section
+        Thread.sleep(2000);
+        Assert.assertTrue(getPlayListDetails().contains(String.valueOf(songsCount())));
+    }
+
+    @Test
+    public void renamePlayList() throws InterruptedException {
+
+        String updatedPlayListMsg = "Updated playlist \"Sample Edited Playlist.\"";
+
+    //  double click on playList Name
+    //  Thread.sleep(2000);
+        hoverPlay();
+    //  doubleClickPlayList();
+    //  Thread.sleep(2000);
+    //  Enter new Name
+    //  enterNewName();
+    //  Thread.sleep(2000);
+    //  Assert.assertEquals(getRenamePlayListSuccessMsg(), updatedPlayListMsg);
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.